### ISSUE TITLE
Delete Icon Of Templates Not Working

### ISSUE THREAD

**Issue by K-Kumar-01** [accordproject/web-components#271]
# Bug Report 🐛
Click on Delete Icon is not working when the cursor hovers over clause templates.
Package: Contract Editor

## Expected Behavior

The template should get deleted.

## Current Behavior

There is no deletion of the template.

## Steps to Reproduce

1. Hover any template
2. Click on delete icon

### Desktop
 - OS: ZorinOs(Ubuntu based)
 - Browser: Chrome
 - Version: Chrome- 88.0.4324.182 (Official Build) (64-bit), Zorin OS 15.3

**Comment by irmerk** [comment]
This is likely an issue with how the icon is used in our Storybook.

**Comment by d-e-v-esh** [comment]
@irmerk I don't think that the edit button works either. That button is supposed to let us edit the contract, isn't it?

**Comment by lokeshn011101** [comment]
I am trying to resolve this!

**Comment by codetheorem** [comment]
Addon action 'delete' is working as it is showing it in the action console.

**Comment by d-e-v-esh** [comment]
@lokeshn011101 What do you think that the problem is?

**Comment by d-e-v-esh** [comment]
I can't seem to find where these 3 functions are defined in the code nor the main props of the application. I am not sure if these three functions are defined yet.
![Code_Q6kl0n9ca2](https://user-images.githubusercontent.com/59534570/110278353-1a528080-7ffd-11eb-9149-7a54f655bc43.png)

The only thing that I could find yet is [this](https://github.com/accordproject/web-components/blob/b6ccb71ca4e103d5c699ad4395653d55597f3919/packages/storybook/src/stories/4-ContractEditor.stories.js#L132). 
![Code_uLx1oZzzTH](https://user-images.githubusercontent.com/59534570/110280172-62bf6d80-8000-11eb-82c0-9e95cc4588cf.png)

All this does is log stuff on the actions console.

**Comment by dselman** [comment]
This is working as designed -- these functions can be supplied by the user of the component. They are not implemented by default because they typically require interaction with other components or the persistence layer of the embedding application.

**Comment by Cronus1007** [comment]
@irmerk Shall I work upon this issue.

**Comment by Cronus1007** [comment]
@irmerk @dselman Before opening the PR the plan that I am intending to follow is :-
- [] First of all the file that I am going to update is [link](https://github.com/accordproject/web-components/blob/master/packages/ui-contract-editor/README.md)
- [] Documenting the role of the user in the Component 
- [] Documenting how to use them in general.

If some sort of addition or deletions are there let me know.

**Comment by irmerk** [comment]
@Cronus1007 I believe that sounds like a good plan.

**Comment by dezfuture** [comment]
I would like to contribute and fix the issue.
Can you guide me on how should I go forward into solving this issue.

**Comment by irmerk** [comment]
@dezfuture as you can see, @Cronus1007 is working on this and has an open PR.

**Comment by Amansingh01-ui** [comment]
I want to contribute on this project and fix this bug.

**Comment by mansigrover09** [comment]
Is this issue resolved?

**Comment by mttrbrts** [comment]
No, but there is a solution in https://github.com/accordproject/web-components/pull/305 if you want to open a new PR. Please consider the feedback given in the thread in #305

**Comment by Utkarshpandey0001** [comment]
i want to work on this issue. if it still exist then can you please assign this to me ?
